chef/cookbooks/barclamp/libraries/barclamp_library.rb

```diff
@@ -39,7 +39,7 @@ def self.list_networks(node)
       def self.get_network_by_type(node, type)
         unless node[:crowbar][:network].nil?
           [type, "admin"].uniq.each do |usage|
-            if found = node[:crowbar][:network].find { |net, data| data[:usage] == usage }
+            if found = node[:crowbar][:network].find { |net, data| net == usage }
```

Comment:
Unrelated to that line (but to the commit and I can't comment on the commit message): Not sure if that file is ever used for testing but in ```crowbar_framework/spec/fixtures/offline_chef/role_crowbar-admin_crowbar_com.json``` is still ```[:crowbar][:network][:admin][:usage]``` and ```[:crowbar][:network][:bmc][:usage]```